ex_webrtc/lib/ex_webrtc/media_event.ex

```diff
@@ -159,7 +156,7 @@ defmodule Membrane.RTC.Engine.Endpoint.ExWebRTC.MediaEvent do
         _other -> @err_invalid_event
       end
     rescue
-      _error in [Protobuf.DecodeError, Jason.DecodeError] ->
+      _error in [Protobuf.DecodeError, Jason.DecodeError, FunctionClauseError] ->
```

Comment:
What about with clause instead of the try catch

Reply:
Since the `Protobuf.decode/2` function raises when something fails, we have to use try-rescue. So if we're doing that, we might as well handle all other decoding errors this way.